src/api/route-services/pipeline-response.js

```diff
@@ -11,19 +11,19 @@ const plotsTableService = new PlotsTablesService();
 
 const plotsInTables = {
   cellSizeDistribution: [
-    'umisInCells',
-    'umisCellRank',
+    'cellSizeDistributionHistogram',
+    'cellSizeDistributionKneePlot',
```

Comment:
Where did these changes come from in your branch?

Reply:
This was the reason why the new plots generated by the pipeline didn't seem to be persisted (refreshing the UI they didn't get loaded from the api).

They were being used to get the plotUuids, but these Uuids seem to be outdated so when I replace them with the ones being used in the rest of the services the persistence works fine.